polaris/hub/client.py

```diff
@@ -866,12 +869,15 @@ def evaluate_competition(
              A `BenchmarkResults` object.
         """
         response = self._base_request_to_hub(
-            url="/v2/competition/evaluate",
+            url=f"/v2/competition/{competition.owner}/{competition.name}/evaluate",
```

Comment:
Thanks for updating this! Totally forgot about it after I updated the hub endpoint. It's just more convenient to follow the conventions already in place there for getting the artifact owner/id out of a request.